src/styles/normalize.scss

```diff
@@ -0,0 +1,13 @@
+body,
+div,
+h2,
+p,
+a {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
You should better set `box-sizing` with `*`
Read more about CSS style inheritance